core/typescript37Action/Dockerfile

```diff
@@ -24,29 +24,17 @@ RUN apt-get update && apt-get install -y \
     imagemagick \
     graphicsmagick \
     unzip \
-    wget \
     && rm -rf /var/lib/apt/lists/* &&\
     mkdir -p /app/action
 RUN cd /app ;\
   npm install -g yarn ;\
   npm install -g typescript@${TYPESCRIPT_VERSION} ;\
-  echo '{"private":true}' >package.json ;\
+  echo '{"private":true}' > package.json ;\
   npm install --save --no-package-lock --production \
-  axios@0.19.0 \
-  async@2.6.1 \
-  aws-sdk@2.401.0 \
-  bluebird@3.5.3 \
-  body-parser@1.18.3 \
-  btoa@1.2.1 \
-  gm@1.23.1 \
   @google-cloud/datastore@4.4.1 \
```

Comment:
@sciabarra if i remove `@google-cloud/datastore@4.4.1 ` the actions don't compile. How is this runtime dependent on @google-cloud/datastore@4.4.1 ?